qase-playwright/src/playwright.ts

```diff
@@ -1,8 +1,189 @@
+import test from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+import { PlaywrightQaseReporter } from './reporter';
+import * as path from 'path';
+
+export const ReporterContentType = 'application/qase.metadata+json';
+const defaultContentType = 'application/octet-stream';
+
+export interface MetadataMessage {
+  ids?: number[];
+  title?: string;
+  fields?: Map<string, string>;
+}
+
+/**
+ * The method should not be used. Use `qase.id` instead
+ *
+ * @param caseId
+ * @param name
+ * @example
+ * test(qase(1, 'test'), async ({ page }) => {
+ *  await page.goto('https://example.com');
+ * });
+ * @returns {string}
+ */
 export const qase = (
   caseId: number | string | number[] | string[],
   name: string,
 ): string => {
+  console.log('qase: The method should not be used. Use `qase.id` instead');
```

Comment:
It won't be clear from the logs which method we're talking about and where to find it. Let's add the actual id here. Something like this:
```suggestion
  console.log(`qase: qase(${caseId}) is deprecated. Use qase.id() and qase.title() inside the test body`);
```